AlphaWallet/Browser/Coordinators/QRCodeResolutionCoordinator.swift

```diff
@@ -282,6 +281,18 @@ extension QRCodeResolutionCoordinator: ScanQRCodeCoordinatorDelegate {
             }
         }
     }
+
+    private func serverFromEip681LinkOrDefault(_ serverInLink: RPCServer?) -> RPCServer? {
+        let server: RPCServer
+        if let serverInLink = serverInLink {
+            return serverInLink
+        }
+        if config.enabledServers.count == 1 {
+            //Specs https://eips.ethereum.org/EIPS/eip-681 says we should fallback to the current chainId, but since we support multiple chains at the same time, we only fallback if there is exactly 1 enabled network
+            return config.enabledServers.first!
+        }
+        return nil
```

Comment:
else? :)

Reply:
@James-Sangalli nah. There are more than one if(s). So they are guard/early return style. No `else` for those.